partials/forms/radio-group.html

```diff
@@ -1,4 +1,5 @@
-<div id="{{key}}-group" class="form-group{{#display}} {{display}}{{/display}}{{#error}} validation-error{{/error}}">
+<fieldset id="{{key}}-group" class="{{#display}}{{display}}{{/display}}{{#error}} validation-error{{/error}}">
+    <legend{{#legend-display}} class="{{legend-display}}"{{/legend-display}}>{{legend}}</legend>
```

Comment:
At present `visuallyhidden` is the only class being applied in PEX, I'm unsure about LSR. Other instances of fieldset also use `form-label-bold`.